docs/source/conf.py

```diff
@@ -0,0 +1,65 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import re
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'git-machete'
+copyright = '2017-2021, Paweł Lipski'
+author = 'Paweł Lipski'
+
+# The full version, including alpha/beta/rc tags
+
+
+def get_last_version_from_release_notes():
+    with open("../../RELEASE_NOTES.md") as file:
+        text = file.read()
+    version_pattern = re.compile(r"[0-9]\.[0-9]\.[0-9]")
+    return version_pattern.search(text).group()
```

Comment:
If building docs will be called with each master build, maybe take `__version__` from `git_machete/__init__.py` directly.  Currently version info in CI process is taken each time from `__init__.py`, i'd rather stick to this also here.

Reply:
Ohh, didn't notice it there, yes, this seems to be significantly simpler, thanks!